scripts/verify-test-resource.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -o nounset # don't allow use of undefined variables
+set -o pipefail # if any single command in a pipe chain exits non-zero, set the exit status to that
+
+# Set environment variable for GitHub action to recognise colours
+export TERM=xterm
+
+# Colours for stdout
+RED=$(tput setaf 1)
+GREEN=$(tput setaf 2)
+WHITE=$(tput setaf 7)
+NC=$(tput sgr0)
+RED_BG=$(tput setab 1)
+
+# Path to test resources
+TEST_RESOURCES_PATH=src/test/resources
```

Comment:
This is dependent on the script being executed from the root of the project. Can be solved like so:

```suggestion
TEST_RESOURCES_PATH="$(git rev-parse --show-toplevel)/src/test/resources"
```